### DISCUSSION TITLE
Timeline table zoom-range below 5 minutes doesnt work

### DISCUSSION THREAD

**Discussion by Grefee** [apexcharts/apexcharts.js#4334] [General]
Hello I'v been trying use this specific chart : ( https://apexcharts.com/react-chart-demos/timeline-charts/advanced/ )
and when you try the 'Selection Zoom' or 'Zoom range' and you get below range 5 minutes .. it doesnt render accordingly
Is it a bug or possible any workaround?

Basically when you get under certain treshold it renders on bottom axis one same timestamp in getTime() format

https://imgur.com/a/nCGKRyI

Thanks in advance

**Comment by ManuelMoeller** [comment]
I have the same problem with the timeline chart. It is also reproducible in the demos (https://apexcharts.com/javascript-chart-demos/timeline-charts/distributed/). Just zoom in below a certain smal time range.
Trying to reset the zoom in this state leads to an empty diagram.

I have tried to limit the range with the `beforeZoom` function, but this does not seem to work properly. Sometimes the event is not triggered, which leads to the situation described above.

Is there a solution to this problem?
